src/services/ldap/LDAPConnectionError.js

```diff
@@ -1,10 +1,14 @@
-const reqlib = require('app-root-path').require;
+const { ApplicationError } = require('../../errors');
 
-const { ApplicationError } = reqlib('src/errors');
+const GENERIC_LDAP_ERROR = {
```

Comment:
Can we please move this error to /errors? @tofixx

Reply:
I guess it is only used by the ldap service. So it makes sense to keep it here.